.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "python.linting.flake8Enabled": true,
+    "python.linting.enabled": true,
+    "python.languageServer": "Pylance",
+    "editor.formatOnSave": true,
+    "files.encoding": "utf8",
+    "files.eol": "\n",
+    "files.insertFinalNewline": true,
+    "files.trimFinalNewlines": true,
+    "files.trimTrailingWhitespace": true,
+    "python.formatting.provider": "black",
+}

```

Comment:
Nitpick but can we please alphabetize these entries? Makes it easier to scan and find one later

Reply:
Sure 👍